osf/external/gravy_valet/translations.py

```diff
@@ -43,20 +47,14 @@ class _LegacyConfigsForWBKey(enum.Enum):
 
 
 def make_ephemeral_user_settings(gv_account_data, requesting_user):
-    if gv_account_data.resource_type == 'authorized-citation-accounts':
-        include_path = ('external_citation_service',)
-        service_key = gv_account_data.get_included_attribute(
-            include_path=include_path,
-            attribute_name='external_service_name'
-        )
+    addon_type = gv_account_data.resource_type.split('-')[1]
+    include_path = f'external_{gv_account_data.resource_type.split('-')[1]}_service',
+    service_wb_key = gv_account_data.get_included_attribute(
+        include_path=include_path,
+        attribute_name='external_service_name' if addon_type == AddonType.CITATION else 'wb_key'
```

Comment:
This is to consistent with current stage configuration. If `wb_key` for citation addons is configured properly, we may replace it with 
```suggestion
        attribute_name='wb_key'
```

Reply:
Will we also need to do this for computing addons?

Reply:
If we set `wb_key` in GV admin, we won't

Reply:
I have two PRs that fix this problem. In the [first one](https://github.com/CenterForOpenScience/gravyvalet/pull/178),  is I added `external_service_name` to the GV ExternalService serializer. And I changed `make_ephemeral_user_settings` and `make_ephemeral_node_settings` in [this PR](https://github.com/CenterForOpenScience/osf.io/pull/10837/files).